WebApiCoreSeed.WebApi.IntegrationTests/Users/UserIntegrationTest.cs

```diff
@@ -0,0 +1,285 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.EF;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.WebApi.IntegrationTests.Fake;
+using WebApiCoreSeed.WebApi.IntegrationTests.Users.TestData;
+using WebApiCoreSeed.WebApi.Models;
+using Xunit;
+
+namespace WebApiCoreSeed.WebApi.IntegrationTests.Users
+{
+    public class UserIntegrationTest
```

Comment:
Let's rearrange the filename, classname and namespace. Start with integration tests for controllers so we should have a `WebApiCoreSeed.WebApi.IntegrationTests.Controllers` namespace. Classname and filename should be `UserControllerTests`.